src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -465,6 +473,23 @@ public function testDecodeEmptyXml()
         $this->encoder->decode(' ', 'xml');
     }
 
+    protected function getXmlPISource()
+    {
+        return '<?xml version="1.0"?>'."\n".
+            '<?xml-stylesheet type="text/xsl" href="/xsl/xmlverbatimwrapper.xsl"?>'.
+            '<?display table-view?>'.
+            '<?sort alpha-ascending?>'.
+            '<response>'.
+            '<foo>foo</foo>'.
+            '<?textinfo whitespace is allowed ?>'.
+            '<bar>a</bar><bar>b</bar>'.
+            '<baz><key>val</key><key2>val</key2><item key="A B">bar</item>'.
+            '<item><title>title1</title></item><?item ignore-title ?><item><title>title2</title></item>'.
+            '<Barry><FooBar id="1"><Baz>Ed</Baz></FooBar></Barry></baz>'.
+            '<qux>1</qux>'.
+            '</response><?instruction <value> ?>'."\n";
```

Comment:
Here is how it could look in case you don't know nowdoc:

```php
<?php
         return <<<'XML'
<?xml version="1.0"?>
<?xml-stylesheet type="text/xsl" href="/xsl/xmlverbatimwrapper.xsl"?>
<?display table-view?>
<?sort alpha-ascending?>
<response>
<foo>foo</foo>
<?textinfo whitespace is allowed ?>
<bar>a</bar><bar>b</bar>
<baz><key>val</key><key2>val</key2><item key="A B">bar</item>
<item><title>title1</title></item><?item ignore-title ?><item><title>title2</title></item>
<Barry><FooBar id="1"><Baz>Ed</Baz></FooBar></Barry></baz>
<qux>1</qux>
</response><?instruction <value> ?>
XML;
```
- no quotes
- no linebreaks
- no concatenation
- but it kind of breaks the indention